rfc/9/comments/2/index.md

```diff
@@ -0,0 +1,25 @@
+# RFC-9: Comment 2
+
+(rfcs:rfc9:comment2)=
+
+## Comment authors
+
+This comment was written by: Joost de Folter, BioImaging-NL
+
+## Conflicts of interest (optional)
+
+None
+
+## Summary
+
+We have implemented a validator and a simple implementation using Zarr-python [validation](https://github.com/jwindhager/ngff-rfc9-zipped-ome-zarr/tree/main/playground/validation).
+
+## Minor comments and questions
+
+- Explicitly state that only Zarr v3 is supported
```

Comment:
OME-NGFF v0.5 is only specified on Zarr v3. Presumbably, if the RFC is accepted would be incorporated into OME-NGFF v0.7 or greater.
https://ngff.openmicroscopy.org/0.5/index.html#storage-format

Do you think the language above needs to be clarified further?

Reply:
Confirmed; now moot. I'll remove. Thanks, @mkitti and @folterj